src/Styles/uploadModal.css

```diff
@@ -0,0 +1,49 @@
+.uploadModal__closeBtn {
+  border-radius: 10px;
+  padding: 20px;
+  border: 2px solid var(--light-blue);
+  cursor: pointer;
+  display: block;
+  margin-bottom: 40px;
+}
+.uploadModal__form {
+  width: 80%;
+  margin: 0 auto;
+  font-family: "Kanit", sans-serif;
```

Comment:
I don't think I have to remind you that if you have to use a font-family declaration more than once, you should put it in a variable

Reply:
I only used once that was why i left it like this

Reply:
Twice actually ... Check again